src/pages/LandingPage/index.vue

```diff
@@ -4,10 +4,4 @@
   </q-page>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
-
-export default defineComponent({
-  name: 'IndexPage',
-});
-</script>
+<script setup></script>
```

Comment:
Is this really needed? According to [docs here](https://vuejs.org/api/sfc-script-setup.html#basic-syntax) you would need to execute "something" every time an instance of the component is created. but this doesn't seem like the case.

Reply:
Is not needed, I confirmed now